front/notificationtemplatetranslation.form.php

```diff
@@ -46,6 +46,10 @@
 $language = new NotificationTemplateTranslation();
 
 $template = new NotificationTemplate();
+if (!isset($_GET["notificationtemplates_id"]) && $_GET["id"] != '') {
+    $language->getFromDB($_GET["id"]);
+    $_GET["notificationtemplates_id"] = $language->fields["notificationtemplates_id"];
+}
 $template->getFromDB($_GET["notificationtemplates_id"]);
 $_SESSION['glpilisturl'][NotificationTemplateTranslation::getType()] = $template->getLinkURL();
```

Comment:
Problem appears in 2 conditions:
1. when template translation form is submitted
2. when clicking on the link generated by the new template translation just created

I figured out that loading the template is only usefull to update the list link in display context, so I think these lines of code should be moved in the `else` block, to not try to load 1 or 2 objects that will not be used in other contexts.